src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -100,12 +100,25 @@ public static GenotypesContext subsetAlleles(final GenotypesContext originalGs,
                 gb.attribute(GATKVCFConstants.STRAND_COUNT_BY_SAMPLE_KEY, newSACs);
             }
 
+            //don't allow NON_REF genotype calls and set PLs to all zeros because we have no idea what's going on here
+            if (gb.make().getAlleles().contains(Allele.NON_REF_ALLELE)) {
```

Comment:
It seems a bit wrong to `make()` `gb` prematurely just to check the alleles.  Perhaps that's a sign that this logic should be moved into `GATKVariantContextUtils.makeGenotypeCall` a few lines up.